### PR TITLE
Adding Ui changes in /stub-generation

### PR DESCRIPTION
**Pull request by Aditya-eddy**
Currently the UI in /stub-generation needs improvement in the following area:

1. The boxes needs to adjusted like as in Homepage of Keploy:
Screenshot from 2024-04-03 09-44-10
2. Space is needed between the social-cloud:
Screenshot from 2024-04-03 09-44-19

please propose any new changes needed.

### CODE REVIEW

**Comment by Aditya-eddy** [comment]
Hey @Hermione2408 i have adjusted the the above changes as:
1.
Screenshot from 2024-04-03 14-20-36
2.
Screenshot from 2024-04-03 14-20-28

File: components/stub-generation/block.tsx [hunk 1]
```diff
@@ -12,24 +18,83 @@ const Item = (props: { title: string, description: string, SvgComponent: Functio
...
+            </h1>
             <div className="flex justify-center w-full">
               <h3 className="h3 text-xl text-gray-600 flex justify-center items-center">
-
                 &nbsp; 👋 Say Hi to 👋 &nbsp;
-    
               </h3>
             </div>
+            <div className="flex text-gray-600 pt-3 pl-6 pr-6">
+              <svg
```
Comment (Hermione2408) [inline components/stub-generation/block.tsx]: Please create separate img for svg and use it that would be more cleaner and Also these arrow placement is not correctly aligned with the text please check
Screenshot from 2024-04-03 18-11-41

File: components/stub-generation/community.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ const SocialCloud: React.FC = () => {
           🐰 Join the Keploy community ✨
         </h2>
       </div>
-      <div className="social-cloud" >
+      <div className="social-cloud gap-2" >
```
Comment (Hermione2408) [inline components/stub-generation/community.tsx]: for small screen you can keep it as it is

**Review by Hermione2408 — CHANGES_REQUESTED** [review]

File: components/stub-generation/block.tsx (hunk 1 above)
Comment (Aditya-eddy) [reply components/stub-generation/block.tsx]: Hey @Hermione2408 just to confirm are you asking to replace the arrow svgs with their respective images?

Comment (Hermione2408) [reply components/stub-generation/block.tsx]: > Hey @Hermione2408 just to confirm are you asking to replace the arrow svgs with their respective images?

you can create a separate file for this svg then use that here as image

File: components/stub-generation/community.tsx (hunk 2 above)
Comment (Aditya-eddy) [reply components/stub-generation/community.tsx]: okay, Thanks!

File: components/stub-generation/block.tsx (hunk 1 above)
Comment (Aditya-eddy) [reply components/stub-generation/block.tsx]: okay!

**Comment by Aditya-eddy** [comment]
hey @Hermione2408 does this looks good?
Screenshot from 2024-04-12 14-33-08

**Review by Hermione2408 — CHANGES_REQUESTED** [review]
Hey @Aditya-eddy the image for the arrow is not same as the one that we are using in main page please check this
Screenshot from 2024-04-15 12-07-21
The placement of the arrows seems a bit off—they're too far outside. They should align with the div above them for a cleaner look.  
Screenshot from 2024-04-15 12-08-59
Also please revert back linting changes

**Comment by Aditya-eddy** [comment]
Does this looks good?
Also I did'nt do any linting changes.
Screenshot from 2024-04-15 14-41-35

**Comment by Hermione2408** [comment]
Please resolve conflicts

**Comment by Aditya-eddy** [comment]
Hey @Hermione2408 i have resolved the conflicts in it!

File: components/pillar-page/block.tsx [hunk 3]
```diff
@@ -42,9 +44,12 @@ export default function ProblemBlocks(props: {
               <h3 className="h3 text-xl text-gray-600 flex justify-center items-center">
                 &nbsp; {props.subtitle} &nbsp;
               </h3>
-            </div>
+            </div> 
+          </div>
+          <div className="flex flex-row justify-center gap-40 text-gray-600 mb-16">
```
Comment (Hermione2408) [inline components/pillar-page/block.tsx]: please decrease the margin-bottom and remove this class from h1 'whitespace-nowrap'

**Review by Hermione2408 — CHANGES_REQUESTED** [review]

File: components/pillar-page/block.tsx (hunk 3 above)
Comment (Aditya-eddy) [reply components/pillar-page/block.tsx]: fixed it!

**Review by Hermione2408 — APPROVED** [review]
LGTM @nehagup

**Review by nehagup — APPROVED** [review]

**Pull request merged**